components/news-letter-input/index.tsx

```diff
@@ -0,0 +1,26 @@
+import React, { FormEvent } from "react";
+
+const NewsLetterInput = () => {
+  function handleSubmit(e: FormEvent) {
+    e.preventDefault();
+  }
+  return (
+    <form onSubmit={handleSubmit} className="flex justify-center mt-8 gap-4">
+      <input
+        type="email"
+        name="soso-email"
+        className="bg-[#ECECEC] w-[25rem] p-4 rounded-lg"
```

Comment:
Lets avoid this, lets create custom colors that match the brands colors in the tailwind-config, e.g `bg-brand-xxxx`